### PR TITLE
fix(rust): should keep import for builtin node modules as it is

### PR DESCRIPTION
**Pull request by hyf0**

### CODE REVIEW

**Comment by hyf0** [comment]
* **#2460** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2460?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @hyf0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: crates/rolldown/tests/rolldown/function/resolve/alias_to_node_builtin_module/artifacts.snap [hunk 1]
```diff
@@ -6,10 +6,10 @@ source: crates/rolldown_testing/src/integration_test.rs
 ## main.js
 
 ```js
-import "node:fs";
+import "shouldRenamedToFs";
```
Comment (IWANABETHATGUY) [inline crates/rolldown/tests/rolldown/function/resolve/alias_to_node_builtin_module/artifacts.snap]: This case is unexpected.

Comment (hyf0) [reply crates/rolldown/tests/rolldown/function/resolve/alias_to_node_builtin_module/artifacts.snap]: Same problem. The case rely on oxc-resolver's alias resoving.

Comment (IWANABETHATGUY) [reply crates/rolldown/tests/rolldown/function/resolve/alias_to_node_builtin_module/artifacts.snap]: let's on hold this.

**Comment by IWANABETHATGUY** [comment]
Now the issues could be solved after https://github.com/rolldown/rolldown/pull/2485 is merged.

**Review by IWANABETHATGUY — APPROVED** [review]

**Pull request merged**